src/metrics/metadata/metadata.go

```diff
@@ -280,21 +282,26 @@ func (metadatas PipelineMetadatas) ApplyOrRemoveDropPolicies() (
 	return result, RemovedIneffectiveDropPoliciesResult
 }
 
-// ApplyCustomTags applies custom M3 tags.
-func (metadatas PipelineMetadatas) ApplyCustomTags() (
-	dropTimestamp bool,
-) {
-	// Go over metadatas and process M3 custom tags.
+// ShouldDropTimestamp applies custom M3 tags.
+func (metadatas PipelineMetadatas) ShouldDropTimestamp(untimedRollups bool) bool {
+	// Go over metadatas and and look for drop timestamp tag.
 	for i := range metadatas {
+		if untimedRollups {
+			for j := range metadatas[i].Pipeline.Operations {
```

Comment:
there's a `Pipeline#IsMappingRule`

Reply:
To use to invert this? i.e. `if !metadatas[i].Pipeline.IsMappingRule()`?  Maybe there should also be a corresponding `Pipeline#IsAnyRollupRules` that simply does the `not` of the `IsMappingRule` call and we could call that?

Reply:
Yeah, invert that. But yeah can create a IsAnyRollupRules method.